actionview/test/template/tag_helper_test.rb

```diff
@@ -348,6 +348,7 @@ def test_class_names
     assert_equal "song play", class_names({ "song": true, "play": true })
     assert_equal "", class_names({ "song": false, "play": false })
     assert_equal "123", class_names(nil, "", false, 123, { "song": false, "play": false })
+    assert_equal "song-play", class_names(song_play: true)
```

Comment:
How should `class_names([ "song_play" ])` behave?

Reply:
I would expect it to return `"song_play"`, and it does. This PR only changes how we process the hash keys.